### DISCUSSION TITLE
Szukanie oceny użytkownika po ID oceny (sale/user-ratings)

### DISCUSSION THREAD

**Discussion by arolznov** [allegro/allegro-api#7951] [Suggestions - resolved]
:bulb: **Sugestia / Suggestions**

Witam, przydałaby się opcja szukania oceny użytkownika po id oceny. Dlatego proponuję albo:
dorobić metodę GET /sale/user-ratings/{ratingId}, która zwróci ocenę lub kod 404 gdy nieistnieje
albo
dorobić parametr ids w metodzie GET /sale/user-ratings aby wyszukiwać oceny po określonym id.

Niekiedy klienci usuwają oceny i jedyną opcją, żeby sprawdzić czy ocena istnieje jest pobranie wszystkich ocen i znalezienie różnic

**Comment by Lukasz-Zurek** [comment]
@arolznov Dziękujemy za Twoją sugestię. Twój wątek przenoszę do zakładki [Discussions](https://github.com/allegro/allegro-api/discussions), dzięki czemu również pozostali użytkownicy będą mogli zagłosować na opisany pomysł rozwoju API w kwestii ocen.

**Comment by MaciejFrackowiak** [comment]
Zaproponowane rozwiązanie mamy już [wdrożone](https://developer.allegro.pl/news/ocena-sprzedazy-dodalismy-zasob-do-pobierania-informacji-o-wskazanej-ocenie-E7keA9KDqfV) w publicznym API.
